lms/djangoapps/commerce/management/commands/create_orders_for_old_enterprise_course_enrollment.py

```diff
@@ -220,6 +220,7 @@ def _sync(self, enrollments_queryset, enrollments_count, enrollments_batch_size)
                 invalid_enrollments += invalid
                 non_paid_enrollments += non_paid
                 new_created_order_numbers += order_numbers
+                time.sleep(1)  # to make sure request per minute could not exceed by 60
```

Comment:
Can make time to sleep configurable from by argument like [this](https://github.com/edx/ecommerce/blob/d9095699a289e2483bced0c0b774f7ce1e1562c2/ecommerce/extensions/order/management/commands/migrate_partner_to_orders.py#L34) ?